Pcap++/header/PcapLiveDevice.h

```diff
@@ -58,26 +58,30 @@ namespace pcpp
-	 * Main capabilities of this class:
-	 * - Get all available information for this network interfaces such as name, IP addresses, MAC address, MTU, etc. This information is taken
-	 * from both libpcap and the OS
-	 * - Capture packets from the network. Capturing is always conducted on a different thread. PcapPlusPlus creates this
-	 * thread when capturing starts and kills it when capturing ends. This prevents the application from being stuck while waiting for packets or
-	 * processing them. Currently only one capturing thread is allowed, so when the interface is in capture mode, no further capturing is allowed.
-	 * In addition to capturing the user can get stats on packets that were received by the application, dropped by the NIC (due to full
-	 * NIC buffers), etc. Stats collection can be initiated by the user by calling getStatistics() or be pushed to the user periodically by
+	 * This class wraps the libpcap capabilities of capturing packets from the network, filtering packets and sending
+	 * packets back to the network. This class is relevant for Linux applications only. On Windows the WinPcapLiveDevice
+	 * (which inherits this class) is used. Both classes are almost similar in capabilities, the main difference between
+	 * them is adapting some capabilities to the specific OS. This class cannot be instantiated by the user (it has a
+	 * private constructor), as network interfaces aren't dynamic. Instances of this class (one instance per network
+	 * interface) are created by PcapLiveDeviceList singleton on application startup and the user can get access to them
+	 * by using PcapLiveDeviceList public methods such as PcapLiveDeviceList#getPcapLiveDeviceByIp()<BR> Main
+	 * capabilities of this class:
+	 * - Get all available information for this network interfaces such as name, IP addresses, MAC address, MTU, etc.
+	 * This information is taken from both libpcap and the OS
+	 * - Capture packets from the network. Capturing is always conducted on a different thread. PcapPlusPlus creates
+	 * this thread when capturing starts and kills it when capturing ends. This prevents the application from being
+	 * stuck while waiting for packets or processing them. Currently only one capturing thread is allowed, so when the
+	 * interface is in capture mode, no further capturing is allowed. In addition to capturing the user can get stats on
+	 * packets that were received by the application, dropped by the NIC (due to full NIC buffers), etc. Stats
+	 * collection can be initiated by the user by calling getStatistics() or be pushed to the user periodically by
 	 * supplying a callback and a timeout to startCapture()
-	 * - Send packets back to the network. Sending the packets is done on the caller thread. No additional threads are created for this task
+	 * - Send packets back to the network. Sending the packets is done on the caller thread. No additional threads are
+	 * created for this task
```

Comment:
Add space to align with bullet points

Reply:
Fixed 61ac4b8

Reply:
clang-format pre-commit fails: https://github.com/seladb/PcapPlusPlus/actions/runs/10299753329/job/28507680248?pr=1516

Reply:
CI fail should be fixed by 903844a